Src/Support/GoogleApis.Auth.DotNet4.Tests/Program.cs

```diff
@@ -0,0 +1,41 @@
+// ***********************************************************************
```

Comment:
Would it be worth having a single copy of this, linked from each of the projects?